Scripts/Modules/MultipleRayInputModule/MultipleRayInputModule.cs

```diff
@@ -386,20 +386,23 @@ private void Select(GameObject go)
 
 		private GameObject GetRayIntersection(RaycastSource source)
 		{
-			GameObject hit = null;
 			// Move camera to position and rotation for the ray origin
 			m_EventCamera.transform.position = source.rayOrigin.position;
 			m_EventCamera.transform.rotation = source.rayOrigin.rotation;
 
-			RayEventData eventData = source.eventData;
+			var camera = U.Camera.GetMainCamera();
```

Comment:
We can probably get away with just adjusting near clip, but I was worried about losing depth precision when increasing the ratio of far/near.

Will add a comment about the fact that we are doing this for world scale purposes